app/code/Magento/Customer/Model/ForgotPasswordToken/GetByToken.php

```diff
@@ -0,0 +1,84 @@
+<?php declare(strict_types=1);
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Customer\Model\ForgotPasswordToken;
+
+use Magento\Customer\Api\CustomerRepositoryInterface;
+use Magento\Customer\Api\Data\CustomerInterface;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Exception\State\ExpiredException;
+use Magento\Framework\Phrase;
+
+/**
+ * Class GetByToken
```

Comment:
Repeating class name in the comment is redundant. Clear responsibility description is sufficient.

Reply:
Cannot agree, but fixed.